felix/bpf/ipsets/ipsets.go

```diff
@@ -57,6 +57,8 @@ type bpfIPSets struct {
 	opRecorder logutils.OpRecorder
 
 	lg *log.Entry
+
+	filterIPSet func(string) bool
```

Comment:
Worth a comment saying what this does.  In particular:
* What does `nil` mean?
* Is `true` "filter in" or "filter out"